tests/regression/71-use_after_free/01-simple-uaf.c

```diff
@@ -0,0 +1,12 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+int main() {
+    int *ptr = malloc(sizeof(int));
+    *ptr = 42;
+
+    free(ptr);
+
+    *ptr = 43; // Should report "Use After Free (CWE-416)"
+    free(ptr); // Should report "Double Free (CWE-415)"
```

Comment:
Maybe we should try annotating with `//WARN` (maybe disabling some of the other warnings that also warn here. To then get warnings for both different cases one would probably have to duplicate it.